### ISSUE TITLE
I get syntax error

### ISSUE THREAD

**Issue by pepe291** [SystemStack/web-crawler#7]
Yeah, that 
![image](https://user-images.githubusercontent.com/30422621/75446549-e499a700-5967-11ea-8197-1c5ee897434f.png)

**Comment by LeviBroadnax** [comment]
It looks like that syntax error is coming from python and not python3, does this error show up in a python3 environment?

**Comment by pepe291** [comment]
> It looks like that syntax error is coming from python and not python3, does this error show up in a python3 environment?

No, if I use Python3 it does not show up anything at all, I-m a noob tbh 
![image](https://user-images.githubusercontent.com/30422621/75475796-b2a13880-5999-11ea-9aa8-5c3f0821b458.png)
